res/layout/new_contact_activity.xml

```diff
@@ -35,16 +35,70 @@
 
         <androidx.legacy.widget.Space
             android:id="@+id/fab_spacer"
+            android:layout_width="match_parent"
+            android:layout_height="40dp"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <Button
+            style="@style/ButtonSecondary"
+            android:id="@+id/qrscan_btn"
             android:layout_width="wrap_content"
-            android:layout_height="28dp" />
+            android:layout_height="wrap_content"
+            android:paddingLeft="40dp"
+            android:paddingRight="40dp"
```

Comment:
```suggestion
            android:paddingLeft="20dp"
            android:paddingRight="20dp"
```

i would use less padding here and also fewer padding atop / abottom of the "or ornament" (2/3rd or so, not totally sure which lines to tweak for that)

that way, things still look good but are are shifted a bit to the top, making it less probable, the address input filed is not covered by the keyboard on smaller devices

Reply:
I reduced the padding of the divider from 50dp to 35dp